scripts/verify-vocabulary.sh

```
@@ -0,0 +1,321 @@
+# at once with file:line positions.
+
+# A UTF-8 locale is load-bearing here, not cosmetic: the en word-boundary
+# check classifies the character adjacent to a token via [[:alpha:]], and
+# Unicode letters (CJK ideographs) must count as word characters while CJK
+# punctuation must not — the exact semantics of \b in the reference
+# implementation. The pwsh twin is UTF-16-native and needs no locale pinning.
+export LC_ALL=C.UTF-8
+set -u
+ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
+# shellcheck source=lib.sh
+source "$ROOT/scripts/lib.sh"
```

Shellcheck (0.11.0)

[info] Not following: lib.sh was not specified as input (see shellcheck -x).

(SC1091)